tests/estimators/online/test_joint_ukf.py

```diff
@@ -266,3 +266,176 @@ def test_joint_ecm() -> None:
         assert vt_sigs[i - 1] >= gaussian_metrics[i] / 100, \
             'Real Vt value is within %d std of prediction only %2.2f%% of the time, rather than %2.0f%%!' % \
             (i, 100 * qt_sigs[i - 1], gaussian_metrics[i])
+
+
+def test_joint_ecm_with_normalization() -> None:
```

Comment:
Not something to change now, but advice for later: let's avoid copying-and-pasting code between these test functions. The code here is mostly identical for the test without normalization, so let's separate the logic that's different (e.g., making the estimator) and re-use the bulkly testing logic at the end.

Reply:
That's very good. I was lazy here; should have made these a fixture. I just wanted to make sure things were working, but you are 100% right. Let's take care of fixing this later, and I will make sure to not mess up with the Dual